cmd/ci-operator-prowgen/main.go

```diff
@@ -292,12 +292,16 @@ func generateCiOperatorPodSpec(info *prowgenInfo, secrets []*cioperatorapi.Secre
 	return ret
 }
 
-func generatePodSpecTemplate(info *prowgenInfo, release string, test *cioperatorapi.TestStepConfiguration) *kubeapi.PodSpec {
+func generatePodSpecOthers(info *prowgenInfo, release string, test *cioperatorapi.TestStepConfiguration) *kubeapi.PodSpec {
```

Comment:
Can you please add a comment about "others" being template+multistage tests? "others" is not descriptive (I could not come up with a better name).

Reply:
Added.

Reply:
Perfect!